server/src/main/scala/org/scassandra/server/cqlmessages/CqlProtocolHelper.scala

```diff
@@ -181,6 +181,21 @@ object CqlProtocolHelper {
     ByteString(header ++ serializeInt(body.length) ++ body)
   }
 
+  /**
+   * Reads an int then that many bytes
+   * @param iterator
+   */
+  def consumeLongBytes(iterator: ByteIterator): Array[Byte] = {
```

Comment:
yep - i was originally not returning the bytes but changed it without updating the name